test/known_issues/test-vm-attributes-property-not-on-sandbox.js

```diff
@@ -1,25 +0,0 @@
-'use strict';
-require('../common');
```

Comment:
There is a new test parallel/test-vm-attributes-on-sandbox.js, which corresponds to the edit of this one.